### ISSUE TITLE
Error: youtube_dl.utils.ExtractorError: Unable to extract info section caused by JSONDecodeError

### ISSUE THREAD

**Issue by Roiinbarr** [ytdl-org/youtube-dl#28591]
## Checklist

-->

- [X ] I'm reporting a broken site support issue
- [X ] I've verified that I'm running youtube-dl version **2021.03.25**
- [X ] I've checked that all provided URLs are alive and playable in a browser
- [X ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X ] I've searched the bugtracker for similar bug reports including closed ones
- [X] I've read bugs section in FAQ

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-u', 'PRIVATE', '-p', 'PRIVATE', 'https://vimeo.com/530495258']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dl version 2021.03.25
[debug] Python version 3.7.4 (CPython) - Windows-10-10.0.19041-SP0
[debug] exe versions: ffmpeg 2021-03-28-git-8b2bde0494-full_build-www.gyan.dev, ffprobe 2021-03-28-git-8b2bde0494-full_build-www.gyan.dev
[debug] Proxy map: {}
[vimeo] Downloading login page
[vimeo] Logging in
[vimeo] 530495258: Downloading webpage
[vimeo] 530495258: Extracting information
ERROR: Unable to extract info section (caused by JSONDecodeError('Extra data: line 1 column 3 (char 2)')); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\extractor\vimeo.py", line 644, in _real_extract
    webpage, 'page config'), video_id)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\extractor\common.py", line 1012, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract page config; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\extractor\vimeo.py", line 665, in _real_extract
    config = json.loads(config)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\json\__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\json\decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 1 column 3 (char 2)
Traceback (most recent call last):
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\extractor\vimeo.py", line 644, in _real_extract
    webpage, 'page config'), video_id)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\extractor\common.py", line 1012, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract page config; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\extractor\vimeo.py", line 665, in _real_extract
    config = json.loads(config)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\json\__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\json\decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 1 column 3 (char 2)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\YoutubeDL.py", line 806, in wrapper
    return func(self, *args, **kwargs)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\YoutubeDL.py", line 827, in __extract_info
    ie_result = ie.extract(url)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\extractor\common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "c:\users\roi\appdata\local\programs\python\python37\lib\site-packages\youtube_dl\extractor\vimeo.py", line 678, in _real_extract
    cause=e)
youtube_dl.utils.ExtractorError: Unable to extract info section (caused by JSONDecodeError('Extra data: line 1 column 3 (char 2)')); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

Everything works fine while trying downloading public videos, while I am unable to download private videos from Vimeo (the username and password are correct - I have verified it).

When I was trying to download public videos I also passed the Username and Password, and it worked fine, the problem is when the video is Private. Its seems that the Logging in works fine but fails at the Extracting Information process.

Would appreciate any solutions how to solve this issue.

If this issue is duplicate please provide the linked issue before closing this issue.

**Comment by benduckwdesign** [comment]
I am having the same exact error with a semi-public video (it is private based on URL, but not restricted beyond that.)

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://vimeo.com/211176625/2162db1250', '--all-subs', '--skip-download', '--yes-playlist', '--verbose', '-o', '%(title)s.%(ext)s']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dl version 2020.12.31
[debug] Python version 3.9.1 (CPython) - Windows-10-10.0.19041-SP0
[debug] exe versions: ffmpeg 4.2.3, ffprobe 4.3.1-2021-01-01-full_build-www.gyan.dev
[debug] Proxy map: {}
[vimeo] 211176625: Downloading webpage
[vimeo] 211176625: Extracting information
{};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={}
ERROR: Unable to extract info section (caused by JSONDecodeError('Extra data: line 1 column 3 (char 2)')); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\extractor\vimeo.py", line 635, in _real_extract
    page_config = self._parse_json(self._search_regex(
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\extractor\common.py", line 1010, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract page config; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose 
flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\extractor\vimeo.py", line 658, in _real_extract
    config = json.loads(config)
  File "C:\Program Files\Python39\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Python39\lib\json\decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 1 column 3 (char 2)
Traceback (most recent call last):
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\extractor\vimeo.py", line 635, in _real_extract
    page_config = self._parse_json(self._search_regex(
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\extractor\common.py", line 1010, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract page config; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose 
flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\extractor\vimeo.py", line 658, in _real_extract
    config = json.loads(config)
  File "C:\Program Files\Python39\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Python39\lib\json\decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 1 column 3 (char 2)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\YoutubeDL.py", line 803, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\YoutubeDL.py", line 824, in __extract_info
    ie_result = ie.extract(url)
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\extractor\common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "C:\Users\miniluigi\AppData\Roaming\Python\Python39\site-packages\youtube_dl\extractor\vimeo.py", line 672, in _real_extract
    raise ExtractorError('Unable to extract info section',
youtube_dl.utils.ExtractorError: Unable to extract info section (caused by JSONDecodeError('Extra data: line 1 column 3 (char 2)')); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update 
 on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

```
I'm not even trying to download the video, just the subtitles.

**Comment by remitamine** [comment]
@benduckwdesign you're using an outdated version.

**Comment by benduckwdesign** [comment]
> 
> 
> @benduckwdesign you're using an outdated version.

It's not outdated. It's for April 1st, not January 4th. I literally uninstalled and reinstalled the python module minutes before I tested it again to update it.

**Comment by benduckwdesign** [comment]
Sorry, must have been running with the wrong python environment. Let me try again with clean install.
